lms/tasks/celery.py

```diff
@@ -0,0 +1,109 @@
+"""Celery app and configuration."""
+
+import logging
+import os
+import sys
+from contextlib import contextmanager
+
+import celery.signals
+from celery import Celery
+from kombu import Exchange, Queue
+from pyramid.scripting import prepare
+
+from lms.app import create_app
+
+LOG = logging.getLogger(__name__)
+
+
+app = Celery("lms")
+app.conf.update(
+    # Without some kind of default here we can't even import the module
+    # in the tests
+    broker_url=os.environ.get(
+        "CELERY_BROKER_URL", "amqp://guest:guest@localhost:5674//"
+    ),
```

Comment:
In the spirit of having a single place to look for settings, should this move to `tox.ini`?